src/components/manage/Widgets/SelectWidget.jsx

```diff
@@ -96,17 +96,17 @@ function getDefaultValues(choices, value) {
       value: 'no-value',
     };
   }
-  if (isObject(value)) {
-    return {
-      label: value.title !== 'None' && value.title ? value.title : value.token,
-      value: value.token,
-    };
-  }
+
   if (isArray(value) && choices.length > 0) {
     return value.map((v) => ({
       label: find(choices, (o) => o[0] === v)?.[1] || v,
       value: v,
     }));
+  } else if (isObject(value)) {
+    return {
+      label: value.title !== 'None' && value.title ? value.title : value.token,
+      value: value.token,
```

Comment:
Notice that choices behave differently when comes as an object (`value.token`), I think that's what Victor was talking about!
Nonetheless, I'm okay with the schema you mentioned as that gonna work with Array of objects passed.